external-crates/move/crates/move-compiler/src/typing/core.rs

```diff
@@ -30,25 +30,36 @@ use crate::{
         *,
     },
     typing::deprecation_warnings::Deprecations,
-    FullyCompiledProgram,
 };
 use known_attributes::AttributePosition;
 use move_ir_types::location::*;
 use move_symbol_pool::Symbol;
 use std::{
     cell::RefCell,
     collections::{BTreeMap, BTreeSet, HashMap},
-    sync::Arc,
 };
 
 //**************************************************************************************************
 // Context
 //**************************************************************************************************
 
-pub struct UseFunsScope {
+pub struct UseFunsScope<'env, 'outer> {
```

Comment:
Most of the changes here are not that interesting, and are just building up the inner context per module member